src/Model/EventCallbackUrl.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Mailjet\LaravelMailjet\Model;
+
+/**
+* https://dev.mailjet.com/email-api/v3/eventcallbackurl/
+* Model for EventCallbackUrl
+*/
+class EventCallbackUrl
+{
+    const EVENT_TYPE_OPEN = 'open';
+    const EVENT_TYPE_CLICK = 'click';
+    const EVENT_TYPE_BOUNCE = 'bounce';
+    const EVENT_TYPE_SPAM = 'spam';
+    const EVENT_TYPE_BLOCKED = 'blocked';
+    const EVENT_TYPE_UNSUB = 'unsub';
+    const EVENT_TYPE_TYPOFIX = 'typofix';
```

Comment:
For all const, I think the best is to implement const into the wrapper and use them in all library. Look at this issue https://github.com/mailjet/mailjet-apiv3-php/issues/82